src/ci/docker/test-various/Dockerfile

```diff
@@ -11,7 +11,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   cmake \
   sudo \
   gdb \
-  xz-utils
+  xz-utils \
+  # for musl
```

Comment:
I think it's fine to leave this out for now, most of the other containers are just soups of dependencies anyway and adding a few extra isn't really gonna hurt us one way or another.